gcloud-java-storage/src/main/java/com/google/gcloud/storage/HttpMethod.java

```diff
@@ -20,5 +20,5 @@
  * Http method supported by Storage service.
  */
 public enum HttpMethod {
-  GET, HEAD, PUT, POST, DELETE
+  GET, HEAD, PUT, POST, DELETE, OPTIONS
```

Comment:
If you look [here](https://cloud.google.com/storage/docs/json_api/v1/buckets) it seems that OPTIONS is considered a valid value as well. I experienced the error while listing buckets of a project of mine, so a bucket with `OPTIONS` as cors method can exist :)

We could add all of them even though I don't think `TRACE` and ``CONNECT` have any use here. Adding all of them until we figure out wich ones are really used will do no harm.